sympy/plotting/tests/test_plot.py

```diff
@@ -383,7 +384,17 @@ def test_plot_and_save_5():
         p[0].only_integers = True
         p[0].steps = True
         filename = 'test_advanced_fin_sum.png'
-        p.save(os.path.join(tmpdir, filename))
+
+        # XXX: This should be fixed in experimental_lambdify or by using
+        # ordinary lambdify so that it doesn't warn. The error results from
+        # passing an array of values as the integration limit.
+        #
+        # UserWarning: The evaluation of the expression is problematic. We are
+        # trying a failback method that may still work. Please report this as a
+        # bug.
+        with ignore_warnings(UserWarning):
+            p.save(os.path.join(tmpdir, filename))
```

Comment:
I'm not sure if there is already an issue for this but I seem to remember a PR that attempted to fix this.

Reply:
The issue is #19996